docs/admin/runmetrics.md

```diff
@@ -139,7 +139,7 @@ indicates the number of page faults which happened since the creation of
 the cgroup; this number can never decrease).
 
 
- - **cache:**  
+ - **cache:**
```

Comment:
do we need that line break? <bullet><bold><break> makes for an odd visual .the after looks more natural.

However, I didn't intentionally remove it. When i changed the weight, my text editor must have gone rogue.